### PR TITLE
Email sends,basic loaders and locations fixes

### PR DESCRIPTION
**Pull request by Ruben-Brits**
- Email sending in all the discussed places
- Loaders for the login and register button
- Location field updates  everywhere

### CODE REVIEW

File: libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.scss [hunk 1]
```diff
@@ -225,4 +225,52 @@ input:focus{
...
+}
+
+.fl.larger{
+    height: 150px !important;
+}
+
+textarea{
+    border: 2px solid var(--ion-color-secondary) !important;
+    padding-left: 12px;
 }
```
Comment (PxttytheThxtty) [inline libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.scss:276]: No Newline

File: libs/login/feature/src/lib/login.component.ts [hunk 2]
```diff
@@ -149,29 +149,34 @@ export class LoginComponent implements OnInit {
       )
       else
       {
+        //this.presentLoadingWithOptions();
```
Comment (PxttytheThxtty) [inline libs/login/feature/src/lib/login.component.ts:152]: Commented Out Code

File: libs/parent-edit-activity/feature/src/lib/parent-edit-activity.component.scss [hunk 3]
```diff
@@ -341,3 +352,17 @@ textarea:focus{
...
+
+ion-radio-group{
+    max-width: 411px;
+    min-width: 411px;
+}
+
+.location-div{
+    min-width: 411px;
+    max-width: 411px;
+}
```
Comment (PxttytheThxtty) [inline libs/parent-edit-activity/feature/src/lib/parent-edit-activity.component.scss:368]: No Newline

File: libs/register/feature/src/lib/register.component.ts [hunk 4]
```diff
@@ -131,7 +130,7 @@ export class RegisterComponent {
       }
       else
       {
-          this.verifyLocation(this.parentRegisterDetailsForm.value.location);
+          //this.verifyLocation(this.parentRegisterDetailsForm.value.location);
```
Comment (PxttytheThxtty) [inline libs/register/feature/src/lib/register.component.ts:133]: Commented Out Ocde

**Review by PxttytheThxtty — DISMISSED** [review]

File: libs/login/feature/src/lib/login.component.ts (hunk 2 above)
Comment (Ruben-Brits) [reply libs/login/feature/src/lib/login.component.ts:152]: Gonna leave it there, This is how we will call the other loader

File: libs/register/feature/src/lib/register.component.ts (hunk 4 above)
Comment (Ruben-Brits) [reply libs/register/feature/src/lib/register.component.ts:133]: removed thanks

**Review by PxttytheThxtty — APPROVED** [review]

**Review by priyo5 — APPROVED** [review]

**Pull request merged**